naresh/src/main/java/com/lftechnology/phpjava/assignmentthree/EvenFibonacci.java

```diff
@@ -0,0 +1,50 @@
+package com.lftechnology.phpjava.assignmentthree;
+
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.Scanner;
+
+/**
+ * Created by naresh on 5/16/16.
+ */
+public class EvenFibonacci implements MenuReader {
+    /**
+     * read the user input for the action
+     */
+    @Override
+    public void scanner() {
+
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter the max value.");
+        String maxValue = scan.nextLine();
+        int max = Integer.valueOf(maxValue);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "Integer.parseInt" for this string-to-int conversion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS2130)